### PR TITLE
Log in With Telegram

### PR DESCRIPTION
**Pull request by akbarjorayev**

### CODE REVIEW

**Review by akbarjorayev — COMMENTED** [review]
Good

File: src/pages/Login.js [hunk 1]
```diff
@@ -189,13 +190,17 @@ function Login() {
...
                 </Button>
               </form>
-              <Typography>yoki</Typography>
-              <Button onClick={loginWithTelegram}>
-                <i
-                  className="fa-brands fa-telegram fa-4x"
-                  style={{ color: '#74C0FC' }}
-                ></i>
-              </Button>
+              {isComputer && (
```
Comment (nazarovctrl) [inline src/pages/Login.js:193]: Shu joyda constani ishlatmasdan
Srazi deviceIscomputer()  ishlatsen Nima boladi?

File: src/pages/Login.js [hunk 2]
```diff
@@ -222,3 +227,9 @@ const InputElement = styled('input')(
 );
 
 export default Login;
+
+function deviceIsComputer() {
```
Comment (nazarovctrl) [inline src/pages/Login.js:231]: Nega funksiyani pasda ochgansanu, 
Const ni tegidan ochsen Nima boladi?

**Review by nazarovctrl — CHANGES_REQUESTED** [review]

File: src/pages/Login.js (hunk 2 above)
Comment (akbarjorayev) [reply src/pages/Login.js:231]: Component kattalashin ketadi

**Review by akbarjorayev — COMMENTED** [review]
Ready to **MERGE** @nazarovctrl

**Review by nazarovctrl — APPROVED** [review]
LGTM

**Pull request merged**